fixtures/entrypoints-rpc-tests/tests/entrypoints.spec.ts

```diff
@@ -820,57 +819,6 @@ describe("entrypoints", () => {
 		});
 	});
 
-	test("should throw if binding to version of wrangler without entrypoints support over HTTPS", async ({
-		dev,
-		isolatedDevRegistryPath,
-	}) => {
-		// Start entry worker first, so the server starts with a stubbed service not
-		// found binding
-		const { url, session } = await dev({
-			"wrangler.toml": dedent`
-			name = "entry"
-			main = "index.ts"
-
-			[[services]]
-			binding = "SERVICE"
-			service = "bound"
-		`,
-			"index.ts": dedent`
-			export default {
-				async fetch(request, env, ctx) {
-					return env.SERVICE.fetch("http://placeholder/");
-				}
-			}
-		`,
-		});
-		let response = await fetch(url);
-		expect(await response.text()).toBe(
-			'[wrangler] Couldn\'t find `wrangler dev` session for service "bound" to proxy to'
-		);
-
-		await writeFile(
-			path.join(isolatedDevRegistryPath, "bound"),
-			JSON.stringify({
-				protocol: "https",
-				mode: "local",
-				port: 0,
-				host: "localhost",
-				durableObjects: [],
-				durableObjectsHost: "localhost",
-				durableObjectsPort: 0,
-				// Intentionally omitting `entrypointAddresses`
-			})
-		);
-
-		// Wait for error to be thrown
-		await waitFor(() => {
-			const output = session.getOutput();
-			expect(output).toMatch(
-				'Cannot proxy to `wrangler dev` session for service "bound" because it uses HTTPS. Please upgrade "bound"\'s `wrangler` version, or remove the `--local-protocol`/`dev.local_protocol` option.'
-			);
-		});
-	});
```

Comment:
This test is removed as the miniflare dev registry does support HTTPS (#10281)

Reply:
Can we change this to be a test that passes with https configured? Why do we need to remove it?